python/sglang/srt/mem_cache/chunk_cache.py

```diff
@@ -68,7 +68,7 @@ def cache_unfinished_req(self, req: Req, token_ids: Optional[List[int]] = None):
         req.last_node = entry
 
     def insert(self):
-        raise NotImplementedError
+        raise NotImplementedError()
```

Comment:
What's the reason for the changes here? https://github.com/sgl-project/sglang/blob/0c1c72a0b409f255a1fcea666705af8140da5f1e/python/sglang/srt/constrained/base_tool_cache.py#L57

Reply:
I think it is recommended to raise `NotImplementedError()` instead of `NotImplementedError`

Reply:
ok May we also change https://github.com/sgl-project/sglang/blob/0c1c72a0b409f255a1fcea666705af8140da5f1e/python/sglang/srt/constrained/base_tool_cache.py#L57